shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1373,8 +1373,16 @@ void WebContents::HandleNewRenderFrame(
   // Set the background color of RenderWidgetHostView.
   auto* web_preferences = WebContentsPreferences::From(web_contents());
   if (web_preferences) {
-    std::string color_name;
-    rwhv->SetBackgroundColor(web_preferences->GetBackgroundColor());
+    web_contents()->SetPageBaseBackgroundColor(
+        web_preferences->GetBackgroundColor());
+
+    // When a page base background color is set, transparency needs to be
+    // explicitly set by calling
+    // RenderWidgetHostOwnerDelegate::SetBackgroundOpaque(false).
+    // RenderWidgetHostViewBase::SetBackgroundColor() will do this for us.
```

Comment:
This comment should mention something about the `transparent: true` BrowserWindow option which is gating here.